image/base/download_phantomjs

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-docker run --rm -it -v `pwd`:/tmp samsaffron/phantomjs cp /usr/bin/phantomjs /tmp
+docker run --rm -it -v '`pwd`:/tmp' samsaffron/phantomjs cp /usr/bin/phantomjs /tmp
```

Comment:
this is semantically different, and incorrect.  `-v` argument is followed by '/HOST-DIR:/CONTAINER-DIR' , and shell needs to execute `pwd`, so it can not be in single quotes.